Formula/influxdb.rb

```diff
@@ -54,9 +56,18 @@ def install
     system "make", "generate"
 
     # Build the CLI and server.
-    ldflags = "-s -w -X main.version=#{version}"
-    system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"influx", "./cmd/influx"
-    system "go", "build", *std_go_args(ldflags: ldflags), "-tags", "assets", "-o", bin/"influxd", "./cmd/influxd"
+    ldflags = %W[
+      -s
+      -w
+      -X main.version=#{version}
+      -X main.commit=#{Utils.git_short_head(length: 10)}
+      -X main.date=#{Time.now.utc.iso8601}
```

Comment:
Won't this produce unreproducible builds? 
Shall we use `ENV["SOURCE_DATE_EPOCH"]` here?

Reply:
Sounds good to me!

Reply:
Yes, but upstream doesn't have reproducible builds either. Just sticking with upstream here. If it's really a concern feel free to open a PR to fix; to me, I don't see much of a benefit if upstream isn't even on board with it.

Reply:
I've done some more reading up on what you suggested, @bayandin. Will open a PR for what you propose when I have a chance.

Reply:
Nice! Thanks for the PR @alebcay!